src/s3ql/oauth_client.py

```diff
@@ -59,20 +68,32 @@ def _parse_response(r):
 
     return r.json()
 
-def main(args=None):
 
-    if args is None:
-        args = sys.argv[1:]
 
-    options = parse_args(args)
-    setup_logging(options)
+def googleDrive(options):
```

Comment:
Please follow the conventions of the existing S3QL code (which in turn mostly follows Python PEP8). This means `CamelCase` only for classes and `under_scores` for everything else. Also, method/function names should typically be verbs so you can tell what they do. From the name, there's no way to tell what the `googleDrive` function does.

Reply:
You are right, pending to refactor